src/cpp/session/projects/SessionProjectContext.cpp

```diff
@@ -477,12 +477,13 @@ void ProjectContext::onDeferredInit(bool newSession)
                             this, _1);
    cb.onUnregistered = bind(&ProjectContext::fileMonitorTermination,
                             this, Success());
+
+   bool hideObjectFiles = prefs::userPrefs().hideObjectFiles();
```

Comment:
What do you think about adding another pref (with no UI) whose value is an array of additional directories to exclude from monitoring? Realize the ideal solution is more along the lines of a `.ignore` file, but as that's unlikely to land in 1.3 it'd be nice to have an escape hatch for cases where something besides the hardcoded list needs to be excluded.

Reply:
That sounds like a good idea to me as well -- it'd be nice to have as an escape hatch.